CMakeLists.txt

```diff
@@ -225,6 +221,28 @@ FetchContent_Declare(
 
 FetchContent_MakeAvailable(fmt pmt ut yaml-cpp vir-simd cpp-httplib)
 
+# Fetch SoapySDR -- needed since the distribution version is incompatible w.r.t. stdlibc++ vs. libc++
+if (CMAKE_CXX_COMPILER_ID MATCHES "(GNU|Clang)") # WIP
+    find_package(SoapySDR CONFIG)
+
+elseif (CMAKE_CXX_COMPILER_ID MATCHES "(Clang|GNU)")
```

Comment:
Will this ever match? It's the same condition as the `if`, isn't it?

Reply:
Yes ... :see_no_evil: 

A left-over from debugging....